theano/gof/op.py

```diff
@@ -622,7 +622,7 @@ def __call__(self, *inputs, **kwargs):
                         detailed_err_msg = utils.get_variable_trace_string(ins)
 
                         raise ValueError(
-                            'Cannot compute test value: input %i (%s) of Op %s missing default value. %s' %
+                            'Cannot compute test value: input %i (%s) of Op %s missing default value. \nStacktrace when the variable was created: %s' %
```

Comment:
Remove this change. That line is added by the new call get_variable_trace_string.